crates/factor-key-value-redis/src/lib.rs

```diff
@@ -1,6 +1,8 @@
+mod redis;
```

Comment:
Same point as above about Azure.

Reply:
I opted to move this to a module named `store` to avoid the stutter of `spin_key_value_x::x`.